Book-A-Desk.Api/ApiDependencyFactory.fs

```diff
@@ -11,12 +11,14 @@ module ApiDependencyFactory =
         getEventStore
         reservationCommandsFactory
         getOffices
+        sendEmailNotification
         =
 
         let createBookingsHttpHandler bearerToken =
             BookingsHttpHandler.initialize
```

Comment:
Very nitpicky detail, feel free to ignore :)

I feel like semantically, it's good to separate services that are used in the handler, vs the reservationCommandsFactory. So the order of parameters would be:

getEventStore
sendEmailNotification
reservationCommandsFactory

Reply:
The underlying topic is that we do the send email functionality for now at the wrong place. As said - I think it should be done by an event handler anyway and then these injections are gone. I would like to avoid polluting the factory atm.

Reply:
Agreed